www/src/utils/CodeMirrorEditor.css

```
@@ -39,3 +39,24 @@
+    padding: 4px 8px;
+    flex-wrap: wrap;
+  }
+
+  .codemirror-wrapper .codemirror-toolbar .codemirror-toolbar-item span {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)